content/nmdc/src/explanation/fair_data.md

```diff
@@ -4,7 +4,7 @@
 
 The NMDC is committed to adhering to [FAIR Guiding Principles](https://www.nature.com/articles/sdata201618) with regard to data management best practices. As part of this commitment, the NMDC has established the [FAIR Microbiome Implementation Network](https://www.go-fair.org/implementation-networks/overview/fair-microbiome/) (IN), a consortium interested in microbiome research that has agreed upon a set of objectives and common vision for implementing FAIR principles in the context of microbiome data science.
 
-The FAIR Microbiome IN seeks to align and synergize the efforts of the NMDC with that of the [GO FAIR]( community, leveraging the [GO FAIR](https://www.go-fair.org/) open and inclusive [ecosystem of Implementation Networks](https://www.go-fair.org/implementation-networks/overview/) for the microbiome research community (e.g., [Chemistry IN](https://www.go-fair.org/implementation-networks/overview/chemistryin/), [Metabolomics IN](https://www.go-fair.org/implementation-networks/overview/metabolomics/), Biodiversites INs). The purpose of the FAIR Microbiome IN is to work with microbiome research communities to:
```

Comment:
These IN links go to pages that say "** This GO FAIR Implementation Network is no longer active. [Visit this webpage for an overview of current Implementation Networks](https://www.go-fair.org/implementation-networks/overview/). **"